VERSION

```diff
@@ -0,0 +1 @@
+cmd/commands/VERSION
```

Comment:
why do we need to symlink here?
Do we want to then do something like:
`export VERSION=$(cat VERSION )`

So then when we're describing release steps we can do something like:
`git tag -a $VERSION`?

Reply:
I thought it would just be easier to find the file. I'm pretty sure conventionally the VERSION file is in the root of the project but unfortunately go embed only lets you embed files in the same directory or a subdirectory (and it doesn't follow symlinks) so the real VERSION file has to be in cmd/commands. Editors like vscode let you edit the symlinked file as if it were the real file though.

Reply:
oh, I see! Okie, then! Thanks for clarifying!